site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,263 @@
+
+However, prerendering does use additional memory and network bandwidth. Care must be taken not to over-prender, at a cost of user resources. Only when there is a high likelihood of the prerender page being navigated to, should the page be prerendered.
+
+See the [Measuring performance](#measuring-performance) section for more information on how to measure the actual performance impact in your analytics.
+
+### Viewing Chrome’s omnibox predictions
+
+For the first two use cases, you can view Chrome’s predictions for URLs in the `chrome://predictors` page:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/Oy3gv9Gyhx0eTKWVtUVo.png", alt="Screenshot of the Chrome Predictors page filtered to show low (grey), medium (amber), and high (green) predictions based on text entered.", width="800", height="560" %}
+
+Green lines indicate confidence high enough to trigger prerendering. In this example typing “s” gives a reasonable confidence (amber), but once you type “sh” then Chrome has enough confidence that you nearly always navigate to [https://sheets.google.com](https://sheets.google.com).
+
+This screenshot was taken in a relatively fresh Chrome install, and filtering out zero confidence predictions, but if you view your own predictors you will likely see considerably more entries, and potentially more characters required to reach a high enough confidence level.
+
+These predictors are also what drive the Omnibox suggested options you may have noticed:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/GcJd2GQ76qdXKbcUb1VD.png", alt="Screenshot of the omnibox 'Typeahead' feature", width="400", height="216.5" %}
+
+Chrome will continually update its predictors based on your typing, and selections.
+
+- For a 50% confidence level (shown in amber), Chrome proactively preconnects to the domain, but does not prerender the page.
+- For a 80% confidence level (shown in green), Chrome will prerender the URL.
+
+### Using the Speculation rules API to prerender pages
+
+For the third prerender option, web developers can insert JSON instructions onto their pages to inform the browser about which URLs would benefit from prerendering:
+
+```html
+<script type="speculationrules">
+{
+  "prerender": [
+    {
+      "source": "list",
+      "urls": ["next.html", "next2.html"]
+    }
+  ]
+}
+</script>
+```
+
+The Speculation rules API is planned to be expanded beyond this simple example with the addition of prefetch, [scores](https://github.com/WICG/nav-speculation/blob/main/triggers.md#scores) (for example, the likelihood of a navigation), and syntax to implement [document rules](https://github.com/WICG/nav-speculation/blob/main/triggers.md#document-rules) (for example, matching `href` patterns). For now, only the above syntax is supported in Chrome.
```

Comment:
Inconsistent capitalization of the API name (should be "Speculation Rules API"). Comb over the post and make these consistent throughout.

Reply:
Actually I was told it should be "Speculation rules API" so switched to that throughout.

Reply:
Gotcha. Seems like an odd choice as far as capitalization goes, but if that's how it's presented, go with it.

Reply:
Hmmm seems it's not consistent. Old article uses "Speculation Rules API" and that seems clearer to me so will go with that.